t/shdict.t

```diff
@@ -999,3 +999,179 @@ number expected, got string
 --- no_error_log
 [alert]
 [crit]
+
+
+
+=== TEST 31: decr int
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        content_by_lua '
```

Comment:
Newly added test cases should use `*_by_lua_block`. Please fix other test cases as well. Thank you.